From d60b772b259d704c1c8aa58154a34dd185aaf7c3 Mon Sep 17 00:00:00 2001
From: Andreas Blaesius <skate4life@gmx.de>
Date: Tue, 4 Apr 2017 14:39:39 +0200
Subject: [PATCH 2/4] [CUSTOM] omap4: define BOARD_CANT_REALLOCATE_OMX_BUFFERS

- in the past this was defined in frameworks/av for omap4 by default

Change-Id: Ia359469c9d2cbe6610096c476bf04e435f537167
---
 BoardConfigCommon.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/BoardConfigCommon.mk b/BoardConfigCommon.mk
index c3be998..4b3aba1 100644
--- a/BoardConfigCommon.mk
+++ b/BoardConfigCommon.mk
@@ -30,6 +30,7 @@ BOARD_EGL_WORKAROUND_BUG_10194508 ?= true
 TARGET_FORCE_SCREENSHOT_CPU_PATH ?= true
 
 # Camera
+BOARD_CANT_REALLOCATE_OMX_BUFFERS := true
 TARGET_CAMERASERVICE_CLOSES_NATIVE_HANDLES := true
 
 # Global include changes
-- 
2.7.4

